package.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "react-date-time-group",
+  "version": "1.0.0",
+  "description": "Datepicker with optional time",
+  "main": "dist/DateTimeGroup.js",
+  "scripts": {
+    "ci": "npm run lint && npm run coverage",
+    "coverage": "istanbul cover _mocha -- --compilers js:babel/register --require react-tests-globals-setup test/test* && istanbul check-coverage --statements 95 --branches 95 --functions 95 --lines 95",
+    "prepublish": "mkdir -p ./dist && babel ./src/DateTimeGroup.jsx > ./dist/DateTimeGroup.js",
+    "prestart": "ulimit -n 9999",
+    "start": "watchify -t babelify -t brfs doc/example.js -o doc/example-built -v",
+    "test": "mocha --require react-tests-globals-setup --compilers js:babel/register test/test*",
+    "lint": "make-up src test doc"
+  },
+  "browserify": {
+    "transform": ["brfs"]
+  },
+  "author": "Ollie Buck <ollie.buck@holidayextras.com>",
```

Comment:
you can add yourself here too 
` "author": "Ollie Buck <ollie.buck@holidayextras.com>, Cameron Viner <etc>",`